src/raft/client.c

```diff
@@ -35,30 +37,41 @@ int raft_apply(struct raft *r,
 	}
 
 	/* Index of the first entry being appended. */
-	index = logLastIndex(r->log) + 1;
-	tracef("%u commands starting at %lld", n, index);
+	start = logLastIndex(r->log) + 1;
+	tracef("%u commands starting at %lld", n, start);
 	req->type = RAFT_COMMAND;
-	req->index = index;
+	req->index = start;
 	req->cb = cb;
 
+	sm_init(&req->sm, request_invariant, NULL, request_states, "apply-request",
+		REQUEST_START);
+	queue_insert_tail(&r->leader_state.requests, &req->queue);
+
 	/* Append the new entries to the log. */
-	rv = logAppendCommands(r->log, r->current_term, bufs, local_data, n);
-	if (rv != 0) {
-		goto err;
+	index = start;
+	for (unsigned i = 0; i < n; i++) {
```

Comment:
Is the reason of removing `logAppendCommands()` helper is just cleaner code or I'm missing something?

Reply:
It's just a cleanup, yes.